pdfbox/src/main/java/org/apache/pdfbox/multipdf/PDFMergerUtility.java

```diff
@@ -1498,8 +1499,8 @@ private void updateStructParentEntries(PDPage page, int structParentOffset) thro
         List<PDAnnotation> newannots = new ArrayList<>(annots.size());
         annots.forEach(annot ->
         {
-            int structParent = annot.getStructParent();
-            if (structParent >= 0)
+            int structParent = annot.getCOSObject().getInt(COSName.STRUCT_PARENT, Integer.MIN_VALUE); // allow for negative struct parent values
```

Comment:
This doesn't change anything but the default value if STRUCT_PARENT is null. If Integer.MIN_VALUE makes more sense than -1 the getter should be changed rather than this piece of code

Reply:
I intentionally kept the patch to one class and as simple and minimal as possible, e.g. not renaming variables. So feel free to adapt it and let me know if you have any questions.

Reply:
So the question is, do we need to change the default value if the value isn't set?

Reply:
That would be an API change which would break all specific (i.e. positiv or negative equals) checks for -1 in all integrations.
If a one-parameter method that defaults to `Integer.MIN_VALUE` is necessary (IMHO it is not), it would have to be a new one, e.g. `getIntSigned(COSName)` (and similar for array, possibly for other number types).
The only change in this regard that I consider necessary is improving the API documentation of `getInt(COSName)` (and similar methods) to clarify that they cannot be used if the result may be negative.